ext/zlib/tests/zlib_get_coding_type.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+string zlib_get_coding_type ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.6.0") {
+    die('SKIP php version so lower.');
+}
+?>
+--FILE--
+<?php
+$compressionCoding = zlib_get_coding_type();
+$codingType = "get zlib type error";
+    
+if((!$compressionCoding) || ("gzip" == $compressionCoding) || ("deflate" == $compressionCoding)){
```

Comment:
It seems a little odd to check for all 3 return values here? Since the suite is executed on CLI, then this value will always be false unless otherwise defined by a HTTP request header which do not occur when using make test (and zlib.compression must be set to On in php.ini).

Since make test only supports --POST-- variables to be sent, but not modifications of $_SERVER variables, meaning we cannot really get a meaningful value of testing this function unless we somehow can modify $_SERVER. The zlib extension will initialize ZLIBG(compression_coding) at RINIT, which zlib_get_coding_type() uses, meaning we cannot override $_SERVER before calling zlib_get_coding_type(), so the result will always be false.